source/main.tsx

```diff
@@ -42,8 +45,8 @@ function changeValues() {
   }
 }
 
-setInterval(changeValues, 5000);
+setInterval(changeValues, 200000);
```

Comment:
Can you make the interval 5 seconds again?

Reply:
Totally. Please keep in mind SortedTable does not send if a value is updated (yet) so the table won't stay sorted.